gen/golang/embed/statik.go

```diff
@@ -0,0 +1,14 @@
+// Code generated by statik. DO NOT EDIT.
+
+package embed
+
+import (
+	"github.com/rakyll/statik/fs"
```

Comment:
@g5becks in webrpc, we're using a forked version of statik at github.com/goware/statik. There are a number of improvements in goware/statik that are required, and in this case.. this file wouldn't need to exist